<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewRace</class>
 <widget class="QDialog" name="addNewRace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>addNewRace</string>
  </property>
  <widget class="QLabel" name="lapsLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Laps:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lapsInput">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="raceTitleLabel">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">opacity: 1; z-index: 1;</string>
   </property>
   <property name="text">
    <string>Race</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lengthInput">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationInput">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>3</width>
     <height>150</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="locationLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lengthLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Length:</string>
   </property>
  </widget>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>3</width>
     <height>150</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>40</y>
     <width>156</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>260</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>260</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
